src/sentry/static/sentry/app/views/projectSettings/index.jsx

```diff
@@ -93,6 +93,7 @@ const ProjectSettings = createReactClass({
     const rootInstallPath = `${pathPrefix}/install/`;
     const path = this.props.location.pathname;
     const processingIssues = this.state.project.processingIssues;
+    const organization = this.context.organization;
```

Comment:
Hmm I think this isn't used anymore and should be removed. All of its children routes are redirects... I wonder if we should just capture a sentry error if this every gets mounted as a transition.

Reply:
Not all the children are redirects. The `projectSettingsRoutes` function is also interpolated as a child. I could add an IndexRedirect and remove this component entirely.

Reply:
I'll create a ticket for this and we should handle it in a diff PR.

Reply:
https://getsentry.atlassian.net/browse/SEN-418